packaging/osx/sharedhost/scripts/postinstall

```diff
@@ -11,6 +11,8 @@ INSTALL_DESTINATION=$2
 chmod 755 $INSTALL_DESTINATION/dotnet
 
 # Add the installation directory to the system-wide paths
+# But first create the directory if it doesn't exist
+mkdir -p /etc/paths.d
```

Comment:
Also, assuming this is confirmed to work, please port this to release/1.1.0 branch as well.